osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -116,6 +122,7 @@ public string StoredBookmarks
         public string Version { get; set; }
 
         public double StarDifficulty { get; set; }
+        public bool ShouldSerializeStarDifficulty() => false;

```

Comment:
what's this for?

Reply:
The ShouldSerialize* in general, or just this specific case?